README.md

```diff
@@ -15,18 +15,15 @@ Dynamic emulation requires coding two files for each resource model: <resource\>
 If a resource needs to be present at ServiceRoot, the ./api\_emulator/resource_manager.py file need to be modified.
 
 
-
-
 # Swordfish Installation Steps:
 
 1. Create an installation directory to install code, install python and create virtual Environment.
 2. Copy the Redfish-Interface-Emulator files into the installation directory.
 3. Open the Swordfish-API-Emulator files.
   -> copy the swordfish /api_emulator/swordfish files into redfish/api_emulator/redfish directory.
   -> copy the Swordfish-API-Emulator/api_emulator files into Redfish-Interface-Emulator/api_emulator directory(resource_manager.py and util.py files)
-4. After Third step is completed, install all the dependencies and modules needed by Redfish-Interface-Emulator and Swordfish-API-Emulator
-
-
-After completing all the above steps , the Swordfish API Emulator should be fully installed, and it should be possible to run it by entering “python emulator.py” at a command prompt.
+4.install all the dependencies and modules needed by Redfish-Interface-Emulator and Swordfish-API-Emulator
```

Comment:
I believe you'll need spaces after the number and period and before the start of the text for this to format properly.

Also, first character of the text should be capitalized.